apc.php

```diff
@@ -87,7 +87,7 @@ function defaults($d,$v) {
     'SORT1' => '/^[AHSMCDTZ]$/',    // first sort key
     'SORT2' => '/^[DA]$/',          // second sort key
     'AGGR'  => '/^\d+$/',           // aggregation by dir level
-    'SEARCH'    => '~^[a-zA-Z0-9/_.-]*$~'           // aggregation by dir level
+    'SEARCH' => '~^[a-zA-Z0-9/_.\-*\\\]*$~' // search term regex
```

Comment:
Any reason not to allow arbitrary inputs?

Reply:
Arbitrary will pass this validation and preg_match that is used for searching. This has worked before and will now.

The problem are specific regex inputs that will not pass validation. I provided a complex example from our app that we use in cache and have not been able to search it.

Reply:
I just want to say that this PR does not break current behavior in any way - it just improves searching via regex.